lambda/src/index.js

```diff
@@ -206,8 +204,7 @@ const handlers = {
 
 var startHandlers = Alexa.CreateStateHandler(states.START,{
     "Start": function() {
-        this.response.speak(WELCOME_MESSAGE).listen(HELP_MESSAGE);
-        this.emit(":responseReady");
+        this.emit(":ask", WELCOME_MESSAGE, HELP_MESSAGE);
```

Comment:
We want to keep the response builder API.

Like this:
```
this.response.speak(WELCOME_MESSAGE).listen(HELP_MESSAGE);
this.emit(":responseReady");
```
Instead of:
```
this.emit(":ask", WELCOME_MESSAGE, HELP_MESSAGE);
```